src/audio/seqplayer.c

```diff
@@ -10,6 +10,41 @@
 #include "audio/playback.h"
 #include "audio/external.h"
 
+/**
+ * Given that (almost) all of these are format strings, it is highly likely
+ * that they are meant to be used in some sort of printf variant. But I don't
+ * care to try and figure out which function gets which string(s)
+ * So I've place them all here instead.
+ * This means some printf stubs have been commented out to avoid duplication
+ * of some strings (i.e seqplayer_unused_string00)
+**/
+char seqplayer_unused_string00[] = "Audio:Track:Warning: No Free Notetrack\n";
+char seqplayer_unused_string01[] = "SUBTRACK DIM\n";
+char seqplayer_unused_string02[] = "Audio:Track: Warning :SUBTRACK had been stolen by other Group.\n";
+char seqplayer_unused_string03[] = "SEQID %d,BANKID %d\n";
+char seqplayer_unused_string04[] = "ERR:SUBTRACK %d NOT ALLOCATED\n";
+char seqplayer_unused_string05[] = "Error:Same List Add\n";
+char seqplayer_unused_string06[] = "Macro Level Over Error!\n";
+char seqplayer_unused_string07[] = "Macro Level Over Error!\n";
+char seqplayer_unused_string08[] = "WARNING: NPRG: cannot change %d\n";
+char seqplayer_unused_string09[] = "Audio:Track:NOTE:UNDEFINED NOTE COM. %x\n";
+char seqplayer_unused_string10[] = "Audio: Note:Velocity Error %d\n";
+char seqplayer_unused_string11[] = "Error: Subtrack no prg.\n";
+char seqplayer_unused_string12[] = "ERR %x\n";
+char seqplayer_unused_string13[] = "Error: Your assignchannel is stolen.\n";
+char seqplayer_unused_string14[] = "Audio:Track :Call Macro Level Over Error!\n";
+char seqplayer_unused_string15[] = "Audio:Track :Loops Macro Level Over Error!\n";
+char seqplayer_unused_string16[] = "SUB:ERR:BANK %d NOT CACHED.\n";
+char seqplayer_unused_string17[] = "SUB:ERR:BANK %d NOT CACHED.\n";
+char seqplayer_unused_string18[] = "Audio:Track: CTBLCALL Macro Level Over Error!\n";
+char seqplayer_unused_string19[] = "[%2x] \n";
+char seqplayer_unused_string20[] = "Err :Sub %x ,address %x:Undefined SubTrack Function %x";
+char seqplayer_unused_string21[] = "Disappear Sequence or Bank %d\n";
+char seqplayer_unused_string22[] = "Macro Level Over Error!\n";
+char seqplayer_unused_string23[] = "Macro Level Over Error!\n";
+char seqplayer_unused_string24[] = "Group:Undefine upper C0h command (%x)\n";
+char seqplayer_unused_string25[] = "Group:Undefined Command\n";
+
```

Comment:
So, is it correct that, some of these were placed in _reasonable_ spots. But you imported more of them so now they all go up here cause effort to figure out where they all go?

I'm fine with that. Just checking.

Reply:
Yes. The placement of the stubbed print's is almost certainly correct (probably just copied from SM64) but trying to get ALL the strings in the right place while respecting those placements would be a lot of effort. So I opted for the slightly lazy option of commenting them out and moving all the strings to the top of the file.